packages/auth/package.json

```
@@ -8,11 +8,12 @@
   "scripts": {
     "knex": "knex",
     "build:deps": "pnpm --filter openapi build",
-    "build": "pnpm build:deps && tsc --build tsconfig.json",
+    "build": "pnpm build:deps && tsc --build tsconfig.json && pnpm copy-files",
     "clean": "rm -fr dist/",
     "fetch-schemas": "./scripts/get-op-schema.sh",
     "test": "jest --passWithNoTests --maxWorkers=50%",
-    "prepack": "pnpm build"
+    "prepack": "pnpm build",
+    "copy-files": "pnpm fetch-schemas && cp -r ./openapi ./dist"
```

```suggestion
    "copy-files": "cp -r ./openapi ./dist"
```
Otherwise `copy-files` & `build` nuke local modified specs.